src/components/MessageBot/MessageBot.stories.tsx

```diff
@@ -4,7 +4,9 @@ import MessageBot from './MessageBot';
 
 const message: string = 'A bot message';
 
-const html: string = '<b>A bot message</b>';
+const html: string = '<b>A html bot message</b> <br>' +
+    'with a html link to the <a href="https://doc.tock.ai">Tock Documentation Page</a><br>' +
+    'and a clickable string url: github.com/theopenconversationkit/tock-react-kit';
```

Comment:
```
const html: string = '<b>A html bot message</b> <br>' +
    'with a html link to the <a href="https://doc.tock.ai">Tock Documentation Page</a><br>' +
    'and a clickable string url: github.com/theopenconversationkit/tock-react-kit<br>vaenl1.com' +
    '<br><a href="tel:0612345678">tel</a>' +
    '<br><a href="mailto:raoul@gmail.com">mail</a><br />' +
    'Bonjour nom.prenom merci pour votre commande';
```

![image](https://user-images.githubusercontent.com/449763/88415415-c03edb00-cdde-11ea-8581-5c1a4ede2cbc.png)

Reply:
https://www.npmjs.com/package/linkifyjs
https://www.npmjs.com/package/get-urls

Reply:
I replaced the "homemade" regex with linkifyjs.